IntelliTect.TestTools.Data.Test/DatabaseFixture.SeedDatabaseOnInitialization.cs

```diff
@@ -38,13 +38,25 @@ public async Task DatabaseFixtureSeed_DatabaseFixtureReused_SeedDataExists()
     [Fact]
     public async Task DatabaseFixtureSeed_PerformMultipleDatabaseOperations_SeedDataExistsOnce()
     {
+        await _DatabaseFixture.PerformDatabaseOperation(_ => { });
+
         await _DatabaseFixture.PerformDatabaseOperation(context =>
         {
             Assert.Equal(5, context.Persons.Count());
             Assert.False(context.Persons.Any(x => x == null));
         });
+    }
+
+    [Fact]
+    public async Task DatabaseFixtureSeed_DatabaseFixtureWithMultipleContexts_SeedDataExistsOnce()
+    {
+        var dbFixture = new DatabaseFixture<ReadOnlySampleDbContext, SampleDbContext>();
+        dbFixture.SetInitialize<SampleDbContext>(SeedData);
 
-        await _DatabaseFixture.PerformDatabaseOperation(context => { });
+        await dbFixture.PerformDatabaseOperation(context =>
+        {
+            Assert.Equal(5, context.Persons.Count());
+        });
```

Comment:
Should this be async?

```suggestion
        await dbFixture.PerformDatabaseOperation(async context =>
        {
            Assert.Equal(5, await context.Persons.CountAsync());
        });
```